ftw/zipextract/interfaces.py

```diff
@@ -0,0 +1,28 @@
+from zope.interface import Interface
+
+
+class IZip(Interface):
+    """ Marker interface for Zip files
+    """
+    pass
+
+
+class IFile(Interface):
+    pass
+
+
+class IFolderCreator(Interface):
+    pass
+
+
+class IFileCreator(Interface):
+    pass
+
+
+class ObjectCreatorBase():
```

Comment:
Since this is not an interface (it does not subclass `Interface`), I'd move that to another file.

Reply:
done